python/social_bot/envs/simple_navigation.py

```diff
@@ -11,17 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""
-A simple enviroment for navigation.
-"""
+"""A simple enviroment for navigation."""
+
 from collections import OrderedDict
-import gym
-import gym.spaces
 import logging
 import numpy as np
 import os
 import random
 
+import gym
+import gym.spaces
+import gin
```

Comment:
Should we also update the dependency for 'gin'?

Reply:
tf_agents already depends on gin

Reply:
You mean for setup.py?

Reply:
I mean if someone wants to use SocialRobot independently, then the 'gin' dependency is not met.